src/modules/intelligence/application/use_cases/translate_article.py

```
@@ -0,0 +1,245 @@
+
+    @staticmethod
+    def _parse_sections(text: str) -> TranslationContent:
+        """Parse translated text into sections by header."""
+        import re
+        header_map = {
+            "summary": "summary",
+            "pain points": "pain_points",
+            "insights": "insights",
+            "innovations": "innovations",
+        }
+        parts = re.split(r'\n(?=(?:Summary|Pain Points|Insights|Innovations)\s*[:：]\s*)', text, flags=re.IGNORECASE)
```

Ruff (0.15.12)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)